crates/re_viewer/src/misc/tensor_to_gpu.rs

```diff
@@ -331,7 +406,7 @@ fn general_texture_creation_desc_from_tensor<'a>(
     })
 }
 
-fn get_or_create_texture<'a, Err>(
+pub fn get_or_create_texture<'a, Err>(
```

Comment:
(I take this a critique on how cumbersome `texture_manager_2d` is to use right now 🤔 )